src/widgets/CalendarWidget.js

```diff
@@ -70,7 +70,7 @@ export default class CalendarWidget extends InputWidget {
   loadZones() {
     const timezone = this.timezone;
     if (!zonesLoaded() && shouldLoadZones(timezone)) {
-      loadZones(timezone).then(() => this.emit('redraw'));
+      loadZones(timezone).then(() => this.emit('rebuild'));
```

Comment:
Did you need to change this back to ```this.emit('redraw')```?  This event is not being handled currently so it serves no purpose to change it to ```rebuild```

Reply:
Omg, yes of course! I'm already confused and forgot to bring it back..